website/site/content/download/personal-edition/ubuntu-upgrade.md

```diff
@@ -12,8 +12,8 @@ Use the URL of the Ubuntu archive package, `focalboard-server-linux-amd64.tar.gz
 Create and use a clean directory, or delete any existing packages first, then run:
 
 ```
-# Download the new version (e.g. 0.7.0 here, check the release for the latest one)
-wget https://github.com/mattermost/focalboard/releases/download/v0.7.0/focalboard-server-linux-amd64.tar.gz
+# Download the new version (e.g. 0.7.3 here, check the release for the latest one)
```

Comment:
Thanks for staying on top of this @ctlaltdieliet! :) However, since 0.7.3 is a plugin-only release, we should not update the Personal Server download doc - there's no download for Personal Server.

So, please revert the change un ubuntu-upgrade.md, thanks.